### PR TITLE
add virtctl expose documentation

### PR DESCRIPTION
**Pull request by yuvalif**

### CODE REVIEW

**Comment by stu-gott** [comment]
Documents https://github.com/kubevirt/kubevirt/pull/962

File: workloads/virtual-machines/expose-service.md [hunk 1]
```diff
@@ -156,18 +184,5 @@ lbsvc     LoadBalancer   172.30.27.5      172.29.10.235,172.29.10.235   27017:31
...
-It is also possible to use `kubectl expose` to expose the pod of the
-VirtualMachine directly:
-
-```bash
-$ kubectl get pod -l "special=key"
-NAME                                READY     STATUS    RESTARTS   AGE
-virt-launcher-windows2012r2-x9x2f   1/1       Running   0          9m
-$ kubectl expose pod virt-launcher-windows2012r2-x9x2f --port=27017 --target-port=3389 --name=lbsvc --type=LoadBalancer
-```
+port. Note that here the external port here (31829) was dynamically allocated.
```
Comment (rmohr) [inline workloads/virtual-machines/expose-service.md]: This last sentence seems to be out of context now.

Comment (yuvalif) [reply workloads/virtual-machines/expose-service.md]: maybe separate to its own line, and move above the "vinagre" comment?
but i wanted to explain how come we see 31829 in the output

Comment (rmohr) [reply workloads/virtual-machines/expose-service.md]: Looks good how it is. I think I have mixed something up.

File: workloads/virtual-machines/expose-service.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ of service are supported: `ClusterIP`, `NodePort` and `LoadBalancer`. The
 default type is `ClusterIP`.
 
 > **Note**: Labels on a VirtualMachine are passed through to the pod, so simply
-> add your labels for service creation to the vm. From there on it works like
+> add your labels for service creation to the VirtualMAchine. From there on it works like
```
Comment (rmohr) [inline workloads/virtual-machines/expose-service.md]: Typo

**Pull request merged**